docs/keystatic.config.tsx

```diff
@@ -144,6 +144,41 @@ export const componentBlocks = {
     },
     chromeless: false,
   }),
+  embed: component({
+    label: 'Embed',
+    preview: props => <div>{props.fields.embedCode.value}</div>,
+    schema: {
+      mediaType: fields.select({
+        label: 'Media type',
+        options: [
+          { label: 'Video', value: 'video' },
+          { label: 'Tweet', value: 'tweet' },
+        ],
+        defaultValue: 'video',
+      }),
+      embedCode: fields.text({
+        label: 'Embed code',
+        multiline: true,
+      }),
+    },
+  }),
+};
+
+const formatting = {
+  headingLevels: [2, 3],
+  blockTypes: true,
+  listTypes: true,
+  inlineMarks: true,
+} as const;
+
+const images = {
```

Comment:
I should have removed this earlier when we added cloud images, but can you please remove this `images` from both the document types? 

Ideally we don't want any content images stored locally in the repo, we'd prefer them in cloud. Removing this kinda enforces it.

Reply:
Makes sense, cheers 🙌🏻